flight/Modules/Actuator/actuator.c

```diff
@@ -83,13 +86,18 @@ static bool manualControlCommandUpdated = true;
 // set true to ensure they're fetched on first run
 static volatile bool actuator_settings_updated = true;
 static volatile bool mixer_settings_updated = true;
```

Comment:
I considered that too.  It does seem to work on F1, but I did not check how much RAM was left.